src/event.rs

```diff
@@ -455,12 +455,7 @@ pub enum WindowEvent {
     RedrawRequested,
 }
 
-/// Represents the pointer type of a pointer event.
-///
-/// ## Platform-specific
-///
-/// **Wayland/X11:** Can only identify [`Touch`](Self::Touch), any other pointer is identified as
-/// [`Mouse`](Self::Mouse).
```

Comment:
Why was this removed?
I believe this should still be valid, no?

Reply:
It doesn't make any sense given that there's only `Mouse` and `Touch`, basically. And `Unknown` is more of a fallback here.

Reply:
I'm not following. The point is that unlike other backends, these backends can't differentiate between unknown and `Mouse`. E.g. while on every other backend a pen will identify as `Unknown`, on X11 and Wayland it identifies as `Mouse`, this is I believe an important discrepancy.

Reply:
Wayland can not identify pen because it's not implemented(I think the same for X11), so compositor sends it as _mouse_, that's true.

I can bring it back, but if something is emulated as mouse, it's a _mouse_, not any other device or Unknown. Like you don't have a case where something is `Unknown` and acts differently on them. 

I guess on other backends `Unknown` does act differently though.

Reply:
> Wayland can not identify pen because it's not implemented(I think the same for X11), so compositor sends it as _mouse_, that's true.

Well, the documentation described the current state, not any potential one.

> I can bring it back, but if something is emulated as mouse, it's a _mouse_, not any other device or Unknown. Like you don't have a case where something is `Unknown` and acts differently on them.
>
> I guess on other backends `Unknown` does act differently though.

What does that mean?
Why would it be *emulated* if its actually a mouse?
What do you mean with "acts differently"?

Reply:
Tablet input on Wayland right now arrives as mouse, because compositor emulates it, because winit doesn't support tablet input protocol. The same will happen with any other pointer input that could be emulated as mouse.

>What do you mean with "acts differently"

Unknown device isn't a mouse, it may not have buttons, etc. Like tablet(it doesn't have buttons, like on mouse).

Like the thing here is that not the issue with Wayland distinguishing devices, but with system ensures that client understands the input with the set of protocol it binds, so `Unknown` can not happed due to compositor making sure to convert everything to mouse.

Reply:
Okay, so you are saying that `Unknown` is never going to happen on Wayland.
It seems to me that the original documentation is correct and should remain in place then.

Reply:
I'll reword to `Unknown` can not happen, rather _can only identify touch_.

Reply:
Sounds good to me, lets sync up that change with all the other types as well please.

Reply:
Done.

Reply:
It should specifically say that its `Mouse` in this case.

Reply:
I'm not sure it matters that much though? Like if client on Wayland supports only Touch, it'll go only via touch.

Reply:
I would like our documentation to be accurate.
It seems very important to me that users are aware that pen events (or any other unsupported type) are emitted as `Mouse` events instead of `Unknown` on Wayland/X11.

I'm having a hard time understanding where you are coming from here and why you would like to not expose this kind of information to users.

Reply:
>It seems very important to me that users are aware that pen events (or any other unsupported type) are emitted as Mouse events instead of Unknown on Wayland/X11.

I want a bit more vague description of what happens because nothing is defined, if I say that everything is converted to `Mouse` then once some compositor will convert to `Touch`, it'll be _confusing_ as well (and I know compositor that converts even mouse to _touch_ for _reasons_).

```
/// **Wayland/X11:** [`Unknown`](Self::Unknown) can not occur due to system converting them to
/// known variants.
```

I can put something like that, since it's the most accurate thing I know.

Reply:
I see!

How about (just trying to improve the English, not the best myself):
```rust
/// **Wayland/X11:** [`Unknown`](Self::Unknown) device types are converted to known variants by the system.
```

Reply:
yeah, fine as well.